packer/linux/scripts/install-docker.sh

```diff
@@ -33,31 +32,6 @@ sudo curl -Lfs -o /etc/systemd/system/docker.socket "https://raw.githubuserconte
 sudo systemctl daemon-reload
 sudo systemctl enable docker.service
 
-if [ "${MACHINE}" == "x86_64" ]; then
-	echo "Downloading docker-compose..."
-	sudo curl -Lsf -o /usr/bin/docker-compose https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-Linux-x86_64
-	sudo chmod +x /usr/bin/docker-compose
-	docker-compose --version
-elif [[ "${MACHINE}" == "aarch64" ]]; then
-  sudo yum install -y gcc-c++ libffi-devel openssl11 openssl11-devel python3-devel
-
-  # docker-compose depends on the cryptography package, v3.4 of which
-  # introduces a build dependency on rust; let's avoid that for now.
-  # https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst#34---2021-02-07
-  # This should be unpinned ASAP; hopefully docker-compose will offer binary
-  # download for arm64 at some point:
-  # https://github.com/docker/compose/issues/7472
-  CONSTRAINT_FILE="/tmp/docker-compose-pip-constraint"
-  echo 'cryptography<3.4' >"$CONSTRAINT_FILE"
-  echo 'urllib3<2' >"$CONSTRAINT_FILE"
-  sudo pip3 install --constraint "$CONSTRAINT_FILE" "docker-compose==${DOCKER_COMPOSE_VERSION}"
-
-	docker-compose version
-else
-  echo "No docker compose option configured for arch ${MACHINE}"
-  exit 1
-fi
-
```

Comment:
![chefskiss](https://github.com/buildkite/elastic-ci-stack-for-aws/assets/15753101/9015e036-5ae6-4839-8842-49c5ffdef1c2)